### DISCUSSION TITLE
I want to send a PR.

### DISCUSSION THREAD

**Discussion by anoonoll** [algolia/instantsearch#5298] [Q&A] [answered]
I found the code to be fixed on the page in the link below.
I would like to send a PR, do you know which page I should modify?
https://www.algolia.com/doc/api-reference/widgets/pagination/react/

**Comment by dhayab** [comment, marked as answer]
Hi, unfortunately our documentation is closed source, but I'm happy to create a pull request based on your suggested changes. What exactly would you like to see differently?

**Reply by anoonoll** [reply]
Thanks for the reply!
It's not a big deal, but I think the showLast code should be changed as follows
Thank you for your consideration.

Currently
```javascript
<Pagination showLast />
```

After change
```javascript
<Pagination showLast={false} />
```

**Reply by dhayab** [reply]
Thanks for clarifying. We generally document non-default usages of the parameters.

In this case, since `showLast` is defaulted to `false` when not explicitly set, we document the usage when you would want to set `showLast` to `true` instead.

**Reply by anoonoll** [reply]
Oh, I see! I understand now! Thank you for the explanation
